example/lib/main.dart

```diff
@@ -173,82 +174,57 @@ class _WidgetGalleryState extends State<WidgetGallery> {
           minWidth: 200,
           builder: (context, _) {
             return SidebarItems(
-              currentIndex: pageIndex,
-              onChanged: (i) => setState(() => pageIndex = i),
-              scrollController: ScrollController(),
+              currenIdentifier: currentIdentifier,
+              onChanged: (id) => setState(() {
+                currentIdentifier = id;
+              }),
+              scrollController: controller,
```

Comment:
The example fails to compile because this constructor argument does not exist. Additionally, there's something weird going on in your git history: this branch is forked off of `cc-custom`, but then merges that branch back into itself after one commit. I'm guessing the two are related.